### PR TITLE
add avatar_url

### LINKED ISSUE #17: add answer avator
Currently only the article author has an avator.

### PR DESCRIPTION
**Pull request by asukaminato0721**
fix #17 

~~btw the correct spelling is avat**o**r, avatar is~~ <https://en.wikipedia.org/wiki/Avatar_(2009_film)>

the `:not` is available even in IE, so it's safe.

<https://caniuse.com/mdn-css_selectors_not>

### CODE REVIEW

File: src/answer.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ const template = renderTemplate`
 <body style="max-width: 1000px; margin: 0 auto; padding: 0 1em 0 1em;">
     <header>
         <h1><a href="${"url"}">${"title"}</a></h1>
+        <img class="origin" src="${"avatar_url"}" />
```
Comment (frostming) [inline src/answer.ts]: It seems should belong to the `<h2 rel="author">` below
While you are on it, can you adjust the layout as the following?
```
 ┌─────────┐┌──────────────┐
 │         ││              │
 │         ││              │
 │ Avatar  │└──────────────┘
 │         │┌──────────────┐
 │         ││              │
 └─────────┘└──────────────┘
```

Comment (asukaminato0721) [reply src/answer.ts]: it requires some more css to do it with flex, I will try.

Comment (asukaminato0721) [reply src/answer.ts]: image

**Review by frostming — COMMENTED** [review]
Please change the tabs to whitespace(4)

File: src/answer.ts [hunk 2]
```diff
@@ -37,21 +37,52 @@ const template = renderTemplate`
...
+				<div class="top">
+					<h2 rel="author">
+						<a href="${"author_url"}" target="_blank">@${"author"}</a>
+					</h2>
+				</div>
+				<div class="bottom">
+					<p> ${"headline"} </p>
+				</div>
+			</div>
+		</div>
```
Comment (frostming) [inline src/answer.ts]: These lines are still tabs

File: src/answer.ts [hunk 3]
```diff
@@ -37,21 +37,52 @@ const template = renderTemplate`
...
     </style>
 </head>
 <body style="max-width: 1000px; margin: 0 auto; padding: 0 1em 0 1em;">
     <header>
         <h1><a href="${"url"}">${"title"}</a></h1>
-        <h2 rel="author">
-		<a href="${"author_url"}" target="_blank">@${"author"}</a>
-		</h2>
-		<p> ${"headline"} </p>
+		<div class="container">
```
Comment (frostming) [inline src/answer.ts]: use a more meaningful class name for author
like `author` and `author-info`

File: src/answer.ts [hunk 4]
```diff
@@ -37,21 +37,52 @@ const template = renderTemplate`
...
     </style>
 </head>
 <body style="max-width: 1000px; margin: 0 auto; padding: 0 1em 0 1em;">
     <header>
         <h1><a href="${"url"}">${"title"}</a></h1>
-        <h2 rel="author">
-		<a href="${"author_url"}" target="_blank">@${"author"}</a>
-		</h2>
-		<p> ${"headline"} </p>
+        <div class="container">
```
Comment (frostming) [inline src/answer.ts]:
```suggestion
        <div class="author">
```

File: src/answer.ts [hunk 5]
```diff
@@ -37,21 +37,52 @@ const template = renderTemplate`
...
+            <div class="avatar">
+                <img class="origin" src="${"avatar_url"}" />
+            </div>
+            <div class="content">
```
Comment (frostming) [inline src/answer.ts]:
```suggestion
            <div class="author-info">
```

File: src/answer.ts [hunk 6]
```diff
@@ -37,21 +37,52 @@ const template = renderTemplate`
...
+                <div class="top">
+                    <h2 rel="author">
+                        <a href="${"author_url"}" target="_blank">@${"author"}</a>
+                    </h2>
+                </div>
```
Comment (frostming) [inline src/answer.ts]: Remove the wrapper div and apply the style directly on the h2 element, then you don't need the `top` and `bottom` class, just:

```css
.author > * {
    flex: 1
}
```

File: src/answer.ts [hunk 7]
```diff
@@ -37,21 +37,52 @@ const template = renderTemplate`
...
+                <div class="bottom">
```
Comment (frostming) [inline src/answer.ts]: remove the wrapper div as well

**Pull request merged**